src/components/views/rooms/RoomHeader.tsx

```diff
@@ -57,6 +57,7 @@ interface IProps {
     e2eStatus: E2EStatus;
     appsShown: boolean;
     searchInfo: ISearchInfo;
+    coreElementsOnly?: boolean;
```

Comment:
can we get a line or two of comments on this to explain what core elements are please?